src/containers/shared/components/Transaction/Clawback/parser.ts

```diff
@@ -0,0 +1,18 @@
+import { Clawback, ClawbackInstructions } from './types'
+import { TransactionParser } from '../types'
+import { formatAmount } from '../../../../../rippled/lib/txSummary/formatAmount'
+
+export const parser: TransactionParser<Clawback, ClawbackInstructions> = (
+  tx,
+) => {
+  const account = tx.Account
+  let amount = formatAmount(tx.Amount)
+  const holder = amount.issuer
```

Comment:
<img width="1504" alt="image" src="https://github.com/ripple/explorer/assets/35279399/08197fe1-51b3-4e8f-a909-6c6f6696d9ec">

It's designed to avoid an extra parameter. Otherwise, the account that submits the tx is the same as the `issuer` field, which would be duplicated. So we utilize the `issuer` for the holder's account instead. 

it's a bit confusing but i believe there has been a precursor somwhere, written in doc
<img width="835" alt="image" src="https://github.com/ripple/explorer/assets/35279399/6a31e07e-c85a-453b-bee4-78f8c19001df">